presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import analyze_pb2
+import analyze_pb2_grpc
+import common_pb2
+import os
+
+from analyzer import RecognizerRegistry
+from analyzer import ContextEnhancer
+
+loglevel = os.environ.get("LOG_LEVEL", "INFO")
+logging.basicConfig(
+    format='%(asctime)s:%(levelname)s:%(message)s', level=loglevel)
+
+DEFAULT_LANGUAGE = "en"
+
+
+class AnalyzerEngine(analyze_pb2_grpc.AnalyzeServiceServicer):
+    def __init__(self, registry=RecognizerRegistry()):
+        # load all recognizers
+        self.registry = registry
+        self.context_enhancer = ContextEnhancer()
+        registry.load_local_recognizer("predefined-recognizers")
```

Comment:
I know it's my code :) but the name is confusing. Should be load_recognizers_from_path or something like that.